### PR TITLE
Fix type error on signup, fix load more button, fix placeholder image

### LINKED ISSUE #8: TypeError on signup
When I follow all the steps in #7 and I submit the form, then I see:

2019-03-08-231410_1920x1080_scrot

### PR DESCRIPTION
**Pull request by matthib**
Close #8 

----

The type error on signup originated mainly from a non-existent connection to the Clock of Change API server, The connection has been fixed in another issue (same network for Back- and Frontend), nevertheless I fixed the case where the type error appears.

Furthermore I fixed the load more button for the Clock of Change user wall and the placeholder image of reloaded entries.

### CODE REVIEW

File: coc/assets/js/coc.js [hunk 1]
```diff
@@ -344,7 +344,7 @@ window.coc = ((window, document, $) => {
           xhr.setRequestHeader('X-WP-Nonce', cocVars.nonce);
         },
         success: function(resp) {
-          if(resp.success === true){
+          if(resp !== null && resp.hasOwnProperty('success') && resp.success === true){
```
Comment (roschaefer) [inline coc/assets/js/coc.js]: Why checking for `resp.hasOwnProperty('success')` ? :thinking:

File: coc/assets/js/coc.js [hunk 2]
```diff
@@ -426,7 +428,7 @@ window.coc = ((window, document, $) => {
               // got merged with lastname on server - output as firstname for compat with old fs
               let uName = obj.firstname;
               let msg = obj.message;
-              let loadedImg = obj.image === '' ? 'https://human-connection.org/wp-content/plugins/coc/assets/images/coc-placeholder.jpg' : obj.image;
+              let loadedImg = obj.image === '' ? cocVars.homeUrl + '/wp-content/plugins/coc/assets/images/coc-placeholder.jpg' : obj.image;
```
Comment (roschaefer) [inline coc/assets/js/coc.js]: Prefer string interpolation with backticks and `${variable}`

File: coc/assets/js/coc.js (hunk 1 above)
Comment (matthib) [reply coc/assets/js/coc.js]: That was overengineering ;) The property 'success' is always available in the response.
Going to remove it!

File: coc/assets/js/coc.js (hunk 2 above)
Comment (matthib) [reply coc/assets/js/coc.js]: Just for readability, or is there any other reason for that?

Comment (roschaefer) [reply coc/assets/js/coc.js]: No, just readability :slightly_smiling_face:

**Pull request merged**